sancus/stubs/multiplication-stub/leak-mul.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python3
```

Comment:
since your script will be inherently specific to the binary build of the enclave, we may want to include the final compiler-generated .elf files in a separate `bin` folder as well